docs/overview.rst

```diff
@@ -0,0 +1,29 @@
+========
+Overview
+========
+
+Ctwrap is a light-weight batch simulation wrapper written in Python. The wrapper consists of two modules:
+the ``parser`` and the ``simulation`` modules. The ``parser`` module is used to parsed the input file and also
+handles units using `pint <https://pint.readthedocs.io/en/stable/>`_,  while
+the ``simulation`` modules handles running batch operation. The ``simulation`` module consists of two classes:
+the :py:class:`~ctwrap.simulation.Simulation` that wraps a simulation module to be run.
+More details about the module is discussed below and the
+:py:class:`~ctwrap.simulation.SimulationHandler` that handles parameter variations and switches between
+multiple configurations.
+
+The wrapper requires a simulation module, a single run python module, that should consists of three
```

Comment:
Comment that this isn't much different from a conventional script. All that is done is that you 'organize' the code